site/documentation/content/api/command-and-control-pubsub/index.md

```diff
@@ -0,0 +1,178 @@
+If so, the response may also include data representing the result of processing the command. This type of command is
+plainly referred to as a *command* because it represents the default case.
+
+The Command & Control API is defined by means of Google Pub/Sub message exchanges, i.e. a client needs to connect to
+Hono and configure Hono for Google Pub/Sub messaging infrastructure in order to use it and invoke operations of the API
+as described in the following sections. Throughout the remainder of this page we will simply use *Pub/Sub* when
+referring to Google Pub/Sub.
+
+The Command & Control API for Pub/Sub is an alternative to the
+[Command & Control API for Kafka]({{< relref "/api/command-and-control-kafka" >}}) for applications that want to
+use Pub/Sub instead of an Apache Kafka&trade; broker to send command messages to devices.
+
+## Send a One-Way Command
+
+Business Applications use this operation to send a command to a device for which they do not expect to receive a
+response from the device.
+For that, the Business Application connects to *Pub/Sub* and writes a message to the tenant-specific topic
+`projects/${google_project_id}/topics/${tenant_id}.command` where `${google_project_id}` is the ID of the Google Cloud
+project and `${tenant_id}` is the ID of the tenant that the client wants to send the command for.
+
+**Preconditions**
+
+1. A Google Cloud Project is set up with the Pub/Sub API enabled.
+2. The ID of the Google Cloud Project is declared as mentioned in the [Publisher and Subscriber Configuration]({{<
+   relref "/admin-guide/pubsub-config#publisher-and-subscriber-configuration" >}}).
+3. The *Business Application* is authorized to write to the topic.
+4. The topic `projects/${google_project_id}/topics/${tenant_id}.command` exists.
+5. The subscription `projects/${google_project_id}/subscriptions/${tenant_id}.command` exists.
+
+**Message Flow**
+
+1. The *Business Application* writes a command message to the
+   topic `projects/${google_project_id}/topics/${tenant_id}.command` on *Pub/Sub*.
+2. Hono consumes the message through the subscription `projects/${google_project_id}/subscriptions/${tenant_id}.command`
+   from *Pub/Sub* and forwards it to the device, provided that the target device is connected and is accepting commands.
+
+**Message Format**
+
+Metadata MUST be set as Pub/Sub attributes on a message. The following table provides an overview of the attributes the
+*Business Application* needs to set on a one-way command message.
+
+| Name       | Mandatory | Type     | Description                                                   |
+|:-----------|:---------:|:---------|:--------------------------------------------------------------|
+| *deviceId* |    yes    | *string* | The identifier of the device that the command is targeted at. |
+| *tenantId* |    yes    | *string* | The identifier of the tenant that the command is targeted at. |
```

Comment:
is there a particular reason why this is needed? The tenant should be easily determined from the topic name, shouldn't it?

Reply:
no you're right, it can be determined from the topic name. Removed it